site/_sass/menus.scss

```diff
@@ -0,0 +1,333 @@
+	$menu-item-width:                  calc(100% + #{$menu-item-border-width});
+//}
+
+/*
+ * Class Selector: .hmenu
+ *
+ * Horizontal Menus.
+ */
+
+/* Applies to outermost <nav> division. */
+/* Assumes one flex item (<ul>)          */
+nav.hmenu
+{
+	display:             block;
+    position:            relative;
+	margin:              0;
+	width:               100%;
+    line-height:         normal;
+    white-space:         nowrap;
+}
+
+/* Applies to <ul> inside outermost <nav> division */
+ul.hmenu
+{
+	align-content:        center;
+	align-items:          center;
+    margin:               0px;
+	display:              flex;
+	flex-direction:       row;
+	flex-wrap:            wrap;
+	font-family:          Arial, Helvetica, sans-serif;
+	font-size:            $menu-item-font-size;
+	gap:                  0px;
+	justify-content:      center;
+	list-style:           none;
+	padding-inline-start: 0;
+	text-align:           center;
+	text-decoration:      none;
+	width:                auto;
+}
+
+/* Applies to <li> tags within the outermost <nav> <ul> tags */
+.hmenu > li
+{
+	display:             block;
+	border:               $menu-item-border-width outset $menu-item-border-color;
+	margin:               0;
+	padding-top:          $menu-item-vertical-padding;
+	padding-bottom:       $menu-item-vertical-padding;
+	padding-inline:       $menu-item-horizontal-padding;
+	position:             relative;
+	
+	color:                $menu-item-normal-color;
+	background-color:     $menu-item-normal-background-color;
+}
+
+/* Make sure that anchor links are a different color when the mouse is not hovering */
+.hmenu li > a
+{
+    color:                $menu-item-normal-color;
+}
+
+/* Flip the border (to look pressed) and font color to highlight */
+.hmenu li:hover
+{
+	border:              $menu-item-border-width inset $menu-item-border-color;
+    background-color:    $menu-item-hover-background-color;
+}
+
+/* Make sure that anchor tag <a> is using the same color used by li:hover */
+.hmenu > li:hover > a
+{
+	color:               $menu-item-hover-color;
+}
+
+/* Applies to first level submenus and makes submenu invisible */
+.hmenu li > nav
+{
+    display:             none;
+	position:            absolute;
+	border:              0px;
+	margin:              0px;
+	top:                 $menu-item-height;
+	left:                $menu-item-negative-border-width;
+	z-index:             $menu-item-z-index;
+}
+
+/* Control position of first Submenu (which is vertical) and makes block visible */
+.hmenu > li:hover > nav
+{
+	display:             block;
+	border:              0px;
+	margin:              0px;
+	padding:             0px;
+}
+
+/* Flips the border when first level Submenu item has mouse focus */
+/* TODO: Is this necessary, or is it covered by ".hmenu > li:hover > nav */
+.hmenu li:hover > nav li:hover
+{
+	border:              $menu-item-border-width inset $menu-item-border-color;
+	display:             block;
+}
+
```

Comment:
using `display: none` won't work well with screen readers. See the [W3C on menu patterns (Disclosure vs. Menubar)](https://www.w3.org/WAI/ARIA/apg/patterns/menubar/examples/menubar-navigation/)

sorry, i should've caught this sooner, but I really don't have enough experience with `a11y`.